Vester/Private/Template/VesterTemplate.Tests.ps1

```diff
@@ -70,6 +70,7 @@ ForEach ($Test in $TestFiles) {
                 'vCenter'    {$cfg.vcenter.vc}
                 'Datacenter' {$Datacenter}
                 'Cluster'    {$Datacenter | Get-Cluster -Name $cfg.scope.cluster}
+                'DSCluster'  {Get-DatastoreCluster -Name $cfg.scope.dscluster}
```

Comment:
Should the datastore cluster object be piped in from $Datacenter?  If an end user specifies a $Datacenter, I believe with the way this is currently written, you'll get all datastore clusters, not just datastore clusters for a particular datacenter.